### DISCUSSION TITLE
Large list of conversations lags TH

### DISCUSSION THREAD

**Discussion by ja-Bertolin** [protz/thunderbird-conversations#1679] [Q&A]
hi,

I am using TB 78.11, MacOS Catalina

When I have a large number of conversations, the add on provokes some sort of lagging, freezing a bit TB. For instance, also if I create a dozen of emails based on a common one, when you try to review the already sent it lags the system.

Disabling the add on the problem dissapears, it works fine.

Thanks

**Comment by Standard8** [comment]
Presumably this is only a lag whilst the messages are loading when you select them? How significant is the lag, is it less than a second or something more?

Generally it is known that Conversations lags with the larger message threads. It has to load all the information first, and then handle the display of them. I've been working towards improving those and have made some advances in the last few weeks, but there are still significant refactors needed to get to where we can fix all of it.

**Comment by ja-Bertolin** [comment]
Thanks for the answer, the lags is around 3-4 secs and sometimes you need to change the message and going back in order to show it. Thanks for your effort trying to solve it.
